sympy/core/tests/test_symbol.py

```diff
@@ -38,7 +38,6 @@ def test_Dummy_force_dummy_index():
     assert Dummy('d1', dummy_index=2) != Dummy('d2', dummy_index=2)
     d1 = Dummy('d', dummy_index=3)
     d2 = Dummy('d')
-    # not only unlikely, in fact impossible (b/c we keep a list of dummy_index)
```

Comment:
OK, I'm still seeing if I can understand what's going on. Am I right in saying that in *this* case there actually is *no* possibility of clash because the internally generated random index is a 7 digit number whereas `d1`'s is 3. *BUT* if someone had chosen `dummy_index=1000000` there would be a 1 in `8*10**6` (or `9*10**6` if you accept my suggested modification) chance of collision?